app/src/main/java/com/framgia/fsalon/screen/customer/CustomerPresenter.java

```diff
@@ -1,15 +1,40 @@
 package com.framgia.fsalon.screen.customer;
 
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports